Formula/distmap.rb

```diff
@@ -0,0 +1,19 @@
+class Distmap < Formula
```

Comment:
I will call this formula `Distmap3alpha` instead, to make sure that it is not the current distmap, nor a user-definitive one. Maybe you should add a note saying that it is for developmental purposes only....